packages/core/components/FilterDisplayBar/FilterMedallion.tsx

```diff
@@ -63,7 +63,9 @@ export default function FilterMedallion(props: Props) {
         }
     }, [textRef, filters]);
 
-    const operator = filters.length > 1 ? "ONE OF" : "EQUALS";
+    const firstFilterValue = filters[0].value.toString();
+    const operator =
+        filters.length > 1 ? "ONE OF" : firstFilterValue.includes("RANGE") ? "BETWEEN" : "EQUALS";
```

Comment:
I love the ternary operator but nesting it feels like it's pushing the limits of readability. (FilterList too)